### DISCUSSION TITLE
new Quasar project cannot open in IE11

### DISCUSSION THREAD

**Discussion by klhebs** [quasarframework/quasar#10961] [CLI - PWA mode] [answered]
hi after i created a new quasar project, i change the quasar.config.js 
`module.exports = configure(function (ctx) {
  return {
    supportTS: false,
    supportIE:true,
....`
and package.json with 
`  "browserslist": [
    "last 10 Chrome versions",
    "last 10 Firefox versions",
    "last 4 Edge versions",
    "last 7 Safari versions",
    "last 8 Android versions",
    "last 8 ChromeAndroid versions",
    "last 8 FirefoxAndroid versions",
    "last 10 iOS versions",
    "last 5 Opera versions",
    "last 3 IE versions"
  ],`

how can i open a new quasar project / make my quasar project to support ie 11

**Comment by MilosPaunovic** [comment, marked as answer]
There is no IE11 support because Vue 3 does NOT (and will not) support it either.

Check out the red warning card at the top:
https://quasar.dev/start/upgrade-guide#introduction

**Reply by SamCcp** [reply]
how can i install previous version of Quasar to support IE? I've tried installing old versions [@quasar/cli@1.1.2] and [@quasar/cli@1.1.3] but the scaffold project it creates still references to vue 3...

I've uninstalled @quasar/cli and also @quasar/app, tried installing older versions but can't get it working on Vue 2.

Some help is really appreciated.

**Reply by rstoenescu** [reply]
<img width="344" alt="image" src="https://user-images.githubusercontent.com/4665641/139574082-242789b4-07e8-4448-8f93-ea468022b80f.png">

<img width="678" alt="image" src="https://user-images.githubusercontent.com/4665641/139574113-973aebff-df59-46db-a788-6604f04ccc79.png">

**Reply by SamCcp** [reply]
Alright, the command i was looking for is --branch v1
Thanks a lot @rstoenescu, i've read the documentation a thousand times but never noticed the branch option.
Indeed, i tried this and it worked like a charm.
